addons/point_of_sale/static/src/js/Screens/PaymentScreen/PaymentScreen.js

```diff
@@ -195,6 +195,7 @@ odoo.define('point_of_sale.PaymentScreen', function (require) {
                 if (!result) {
                     await this.showPopup('ErrorPopup', {
                         title: 'Error: no internet connection.',
+                        // TODO: Ask Payment team.
                         body: error,
```

Comment:
Hello @pimodoo ,

We added a new eslint test to detect undeclared variables on JS, and we will need a little help to make the PR green.

In this specific case, can we remove this line ? Or it lacks a try catch ? 

Thanks for your help

Reply:
@pimodoo I added a comment line to skip this line from the test, like that you can create a task to fix this undeclared variable issue on your pipe. Just don't forget to remove this comment-skip line once the issue is solved. Thanks for your help.